data-prepper-api/build.gradle

```diff
@@ -23,9 +23,9 @@ dependencies {
 jacocoTestCoverageVerification {
     dependsOn(jacocoTestReport)
     violationRules {
-        rule { //in addition to core projects rule - this one checks for 100% code coverage for this project
+        rule {
             limit {
-                minimum = 1.0
+                minimum = 0.9
```

Comment:
We shouldn't lower this. Add unit tests as necessary.

Reply:
i dont have any changes in data-prepper-api in my PR, but it seems to have reduced coverage due to my change. Will address it ...